aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -154,6 +167,12 @@ struct TORCH_API DispatchKeyExtractor final {
         for (const at::Tensor& tensor : ivalue.toTensorList()) {
           ks = ks | tensor.key_set();
         }
+      } else if (C10_UNLIKELY(ivalue.isSymIntList())) {
+        for (const auto& elt : ivalue.toListRef()) {
+          if (elt.isSymInt()) {
+            ks = ks | elt.toSymNodeImpl()->key_set_;
```

Comment:
Why isn't this unowned, and don't you have to check it's heap allocated?

Reply:
Oh it is unowned, just need to rename the method...

You don't have to check its heap allocated because AFAIK IValues tagged with the Tag::SymInt are always heap allocated, otherwise they would've been tagged with Tag::Int.
```cpp
  IValue(const c10::SymInt& i) {
    if (auto mi = i.maybe_as_int()) {
      tag = Tag::Int;
      payload.u.as_int = *mi;
    } else {
      tag = Tag::SymInt;
      payload.u.as_intrusive_ptr = i.toSymNode().release();
    }
  }
```

Reply:
Add a comment about this